site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/toolGroupMultipleDisplay.vm

```diff
@@ -150,6 +150,49 @@
           </label>
       </div>
     </div>
+    <div class="lessonsSubNavToggleArea" style='display: none'>
+      <p>$tlang.getString("sinfo.lessonbuildersubnav.allowForSite")</p>
+      <div class="onoffswitch">
+          <input type="checkbox" name="isLessonsSubNavEnabledForSite" class="onoffswitch-checkbox" id="isLessonsSubNavEnabledForSite" #if($isLessonsSubNavEnabledForSite)checked="checked"#end>
+          <label class="onoffswitch-label" for="isLessonsSubNavEnabledForSite">
+              <span class="onoffswitch-inner"></span>
+              <span class="onoffswitch-switch"></span>
+          </label>
```

Comment:
This on/off isn't accessible but we can switch both this one and the MathJax one after committed